spec/models/room_spec.rb

```diff
@@ -30,41 +38,41 @@
       room.slug_it_up
       expect(room.slug).to eq(room.id.to_s)
     end
-  end
 
-  it "must have at least one choice" do
-    room = Room.new(name: "test")
-    expect(room).not_to be_valid
-    room.choices.build(title: "trivial choice")
-    room.choices.build(title: "other trivial choice")
-    expect(room.save).to be
-    expect(room.choices.count).to eq(2)
-  end
+    it "must have at least one choice" do
+      room = Room.new(name: "test")
+      expect(room).not_to be_valid
+      room.choices.build(title: "trivial choice")
+      room.choices.build(title: "other trivial choice")
+      expect(room.save).to be
+      expect(room.choices.count).to eq(2)
+    end
 
-  context("state attribute") do
-    it "has three possible states" do
-      pending = Room.new(state: "pending")
-      active = Room.new(state: 1)
-      closed = Room.new(state: :closed)
+    context("state attribute") do
+      it "has three possible states" do
+        pending = Room.new(state: "pending")
+        active = Room.new(state: 1)
+        closed = Room.new(state: :closed)
 
-      expect(pending.state).to eq("pending")
-      expect(active.state).to eq("active")
-      expect(closed.state).to eq("closed")
-    end
+        expect(pending.state).to eq("pending")
+        expect(active.state).to eq("active")
+        expect(closed.state).to eq("closed")
+      end
 
-    it "has a display state for pretty factor" do
-      expect(Room.new.display_state).to eq("Pending")
-      expect(Room.new(state: "active").display_state).to eq("Active")
-      expect(Room.new(state: "closed").display_state).to eq("Closed")
-    end
+      it "has a display state for pretty factor" do
+        expect(Room.new.display_state).to eq("Pending")
+        expect(Room.new(state: "active").display_state).to eq("Active")
+        expect(Room.new(state: "closed").display_state).to eq("Closed")
+      end
 
-    it "has a display state which changes depending on whether it is being viewed" do
-      pending = Room.new
-      active = Room.new(state: "active")
-      closed = Room.new(state: "closed")
-      expect(pending.display_current_state).to eq("Active")
-      expect(active.display_current_state).to eq("Active")
-      expect(closed.display_current_state).to eq("Closed")
+      it "has a display state which changes depending on whether it is being viewed" do
+        pending = Room.new
+        active = Room.new(state: "active")
+        closed = Room.new(state: "closed")
+        expect(pending.display_current_state).to eq("Active")
+        expect(active.display_current_state).to eq("Active")
+        expect(closed.display_current_state).to eq("Closed")
+      end
     end
   end
 end
```

Comment:
I don't think it has any problems. It would blow up if it was missing an `end`, and when I run my auto-indentor on it it shows no errors and no changes to indentation. I think it's fine.